### DISCUSSION TITLE
Alpine x-mask not working with wire:model

### DISCUSSION THREAD

**Discussion by devinvb** [livewire/livewire#6911] [7. Bugs (archived)] [answered]
### Livewire version

v3.0.5

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Livewire component with public string property:
```
public string $test = '';
```
2. In the Blade view an input with Alpine x-mask property (example from alpine docs).
```
<div x-data>
    <input x-mask="99/99/9999" placeholder="MM/DD/YYYY">
</div>
```
Masking now works as expected.

3. Wire model input to the public property
```
<div x-data>
    <input wire:model="test" x-mask="99/99/9999" placeholder="MM/DD/YYYY">
</div>
```
Now the masking doesn't work when typing. The mask is however applied on blur, but this is not expected behaviour.
When combining with `wire:model.live`, the applied mask on blur also doesn't sync to the controller. 

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [x] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by Capati** [comment]
Related
https://github.com/alpinejs/alpine/discussions/3748#discussioncomment-7584447

**Comment by Capati** [comment, marked as answer]
For a workaround, applying a small debounce like `wire:model.debounce.5ms` can fix, but I don't know how reliable is.
